plugins/backend/catalog-backend-module-okta/src/providers/OktaGroupEntityProvider.ts

```diff
@@ -64,17 +68,30 @@ export class OktaGroupEntityProvider extends OktaEntityProvider {
         parentKey: string;
         key?: string;
       };
+      schedule?: TaskRunner;
+      scheduler?: PluginTaskScheduler;
     },
   ) {
-    const accountConfig = getAccountConfig(config);
+    if (!options.schedule && !options.scheduler) {
+      throw new Error('Either schedule or scheduler must be provided.');
+    }
+
+    const accountConfig = getAccountConfig(config, DEFAULT_PROVIDER_ID);
+
+    const taskRunner =
+      options.schedule ??
+      options.scheduler!.createScheduledTaskRunner(accountConfig.schedule!);
```

Comment:
Can we schedule when the startup hook has completed? Is that possible with the new backend system?

Reply:
Not a hook but it's configurable: https://github.com/backstage/backstage/blob/ddd67399de0c3ef2e847af90f8e1636f122d972a/packages/backend-tasks/src/tasks/types.ts#L126